docs/quick-start-docker.md

```diff
@@ -102,7 +101,6 @@ In the metrics tab you should see a bunch of metrics already in the root hierarc
 * `metrictank.usage`: usage metrics reported by metrictank.  See
 [Usage reporting](https://github.com/raintank/metrictank/blob/master/docs/usage-reporting.md)
 It may take a few minutes for the usage metrics to show up.
-* `stats`: these are metrics coming from graphite-api, aggregated by statsdaemon and sent back to metrictank every second.
```

Comment:
we should add a note here that this won't show up until statsdaemon receives some stats, there's only a few non-obvious cases in which this happens (user adds graphite-direct datasource, user sends to statsdaemon manually, or user does requests to MT which MT needs graphite for)